Update readme

khdc-me opened the pull request:
Updated README by adding welcoming text, a short 'about us' description, and attempting to logically tie the various docs/resources in this repo.

khdc-me committed:
Revised README.md

khdc-me committed:
Update README.md

m1yag1 commented on README.md:
```diff
@@ -1,2 +1,44 @@
+# About pyslackers
+pyslackers is a rapidly growing, global community of Python enthusiasts whose
+members possess a wide range of familiarity with the language - from those who
+are just beginning to learn it, to those who have been using it for far too long.
```
You may want to include our mission statement somewhere in here:

`PySlackers is dedicated to building and enabling the global Python community by providing a safe and inclusive place to learn, share, and grow as a Pythonista.`

khdc-me committed:
Update README.md

Added mission statement and switched 'pyslackers' to PascalCase throughout doc.
